src/Transfer/EzPlatform/Adapter/EzPlatformAdapter.php

```diff
@@ -52,6 +53,8 @@ class EzPlatformAdapter implements TargetAdapterInterface, LoggerAwareInterface
      */
     protected $contentTypeManager;
 
+    protected $languageManager;
```

Comment:
I think this should have been moved to `Transfer\EzPlatform\Repository\ObjectService` (see line 72). The same applies for `$contentTypeManager`.